Dockerfile.elixir

```diff
@@ -0,0 +1,16 @@
+FROM rust:1.49.0 AS build
+
+COPY . /usr/src/gleam-src
+
+RUN set -xe \
+        && cd /usr/src/gleam-src \
+        && make install \
+        && rm -rf /usr/src/gleam-src
+
+# Gleam fixed to latest Elixir at time of release
+FROM elixir:1.11.3
```

Comment:
I think the best thing to do here is to have the elixir version that we are building upon fixed. It will be part of the release process to upgrade this value if we want.

I think having this step is explicit, because it might be confusing to use elixir:latest as if `gleam:0.13.2` is on `elixir:1.11.3` it makes sense, in my mind, to stay on `elixir:1.11.x` until the next moving gleam up to `0.14.0`